config/rbac/leader_election_role_binding.yaml

```diff
@@ -1,12 +1,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+kind: ClusterRoleBinding
```

Comment:
why we need to use ClusterRole and ClusterRoleBinding for leader-election? it is namespaced operation

Reply:
kubebuilder generates this automatically. please see the https://github.com/percona/percona-server-mysql-operator/blob/2c86dbff36e90d558e8280b5fded892aff21449a/deploy/rbac.yaml for the end result